Daniel-Bennett777/PersonalPlannerAPI
# added new event functionality

to test** GIT FETCH ** 

this branch
-git checkout into main. `git fetch` then choose this branch and then go into the model event.py

-check for changes regarding the date in the events.view.

here is the new body request example at `http://localhost:8000/events` here is an EXAMPLE body request in postman `{
  "category": 1,
  "title": "punching Event",
  "description": "This is a sample event description.",
  "event_date": "2024-05-26",
  "event_time": "12:00:00",
  "city": "Sample City",
  "state": "Sample State",
  "address": "123 Sample St",
  "zipcode": 12345
}`
